NuKeeper/Engine/Packages/PackageUpdater.cs

```diff
@@ -79,7 +84,12 @@ public class PackageUpdater : IPackageUpdater
                 Body = CommitWording.MakeCommitDetails(updates)
             };
 
-            await _github.OpenPullRequest(repository.Pull, pr);
+            var createdPullRequest = await _github.OpenPullRequest(repository.Pull, pr);
+
+            if (_modalSettings.Labels.Any())
+            {
+                await _github.AddLabelsToIssue(repository.Pull, createdPullRequest.Number, _modalSettings.Labels);
```

Comment:
I would have put this `AddLabelsToIssue` call inside `OpenPullRequest`

Reply:
I had it like this first and it felt a bit... Odd. As in, the REST API for GitHub actually allows it in one call, it's just OctoKit being odd and not handling this, so it makes sense to "fix" it in IGithub... but then it's also putting more "opinion" into the driver instead of, well, PackageUpdater. I can change this back to be in Octokit driver if you like it better.

Reply:
I would prefer that, yes. I think the GitHub driver can be a bit opinionated, it's not totally general purpose, it just has to serve NuKeeper purposes. But it should cope with null/empty labels, I